### DISCUSSION TITLE
Error while installing plugin netbox-documents - relation "netbox_documents_sitedocument" already exists

### DISCUSSION THREAD

**Discussion by AustA549** [netbox-community/netbox#17642] [Help Wanted!] [answered]
Hi,

I'm trying to install the netbox-documents plugin in Netbox and received below error. Did anyone successfully installed the netbox-documents plugin and if anyone encountered the same error. 

Error was received after executing this command  (venv) $ python manage.py migrate

----------------------------------------------------------
Python version: 3.11.6
NetBox version: 4.0.7
Plugins: 
  netbox_documents: 0.7.0
----------------------------------------------------------

Plugin was installed but getting this error in the CLI logs:

File "/opt/netbox/venv/lib64/python3.11/site-packages/django/db/backends/utils.py", line 103, in _execute
return self.cursor.execute(sql)
^^^^^^^^^^^^^^^^^^^^^^^^
File "/opt/netbox/venv/lib64/python3.11/site-packages/psycopg/cursor.py", line 97, in execute
raise ex.with_traceback(None)
django.db.utils.ProgrammingError: relation "netbox_documents_sitedocument" already exists

Then I tried to upload a test document but I received another error below:

<class 'django.db.utils.ProgrammingError'>

column "external_url" of relation "netbox_documents_circuitdocument" does not exist
LINE 1: ...pdated", "custom_field_data", "name", "document", "external_...
                                                             ^

**Comment by jeremystretch** [comment, marked as answer]
Please raise any issues installing a NetBox plugin with the maintainer(s) of that plugin.

**Comment by AustA549** [comment]
Hi Jeremy,

Thank you for the quick response.

I was able to figure out the issue earlier and was able to do some
workaround. Looks like there is some sequence and database table validation
issue they need to fix in the plugin.
